kb/pattern/scs/phrase_with_template.scs

```diff
@@ -0,0 +1,17 @@
+@pattern = [* 
+	_message _<- concept_message;
+	_-> rrel_entity:: _entity;;
+	_entity _<- concept_your_class_identifier;
+		_=> nrel_main_idtf: [ ]
+			(*
+				=> nrel_main_idtf: [_id];
+				_<- lang_ru;;
+			*);;
+*];;
+
+concept_phrase_about_your_theme
```

Comment:
```suggestion
concept_phrase_with_template_about_your_theme
```
Need to separete this entity with `concept_phrase_about_your_theme` from `phrase.scs`

Reply:
Fix in gwt too, please